client/state/ui/guided-tours/contexts.js

```diff
@@ -133,6 +134,16 @@ export const hasSelectedSiteDefaultSiteTitle = state => {
 };
 
 /**
+ * Returns true if user has just pasted something from Google Docs.
+ *
+ * @param {Object} state Global state tree
+ * @return {Boolean} True if user has just pasted something from Google Docs, false otherwise.
+ */
+export const hasUserPastedFromGoogleDocs = state => {
+	return getLastAction( state ).type === EDITOR_PASTE_EVENT_FROM_GDOCS;
```

Comment:
To be safe, could you alter `getActionLog` to fall back to `false`? i.e.

```js
( state ) => last( state.ui.actionLog ) || false
```

that way we don't run the risk of throwing TypeError if the actionLog is empty and we're accessing a property directly:

```js
getLastAction( state ).type // evals to
<- null.type // TypeError
-> false.type // safe
```